packages/esm-patient-banner-app/src/banner-tags/print-identifier-sticker.component.tsx

```diff
@@ -0,0 +1,178 @@
+        case 'other':
+          return t('other', 'Other');
+        case 'unknown':
+          return t('unknown', 'Unknown');
+        default:
+          return gender;
+      }
+    };
+
+    const identifiers =
+      patient?.identifier?.filter(
+        (identifier) => !excludePatientIdentifierCodeTypes?.uuids.includes(identifier.type.coding[0].code),
+      ) ?? [];
+
+    return {
+      id: patient?.id,
+      photo: patient?.photo,
+      name: patient ? displayName(patient) : '',
+      dateOfBirth: patient.birthDate,
+      age: age(patient?.birthDate),
+      gender: getGender(patient?.gender),
+      address: patient?.address,
+      identifiers: identifiers?.length ? identifiers.map(({ value, type }) => ({ value, type })) : [],
+    };
+  }, [patient, t, excludePatientIdentifierCodeTypes?.uuids]);
+
+  const handleBeforeGetContent = useCallback(() => {
+    return new Promise((resolve) => {
+      if (patient && headerTitle) {
+        closeModal();
+        onBeforeGetContentResolve.current = resolve;
+        setIsPrinting(true);
+        const printStyles = `@media print { @page { size: ${printIdentifierStickerSize}; } }`;
+
+        const style = document.createElement('style');
+        style.type = 'text/css';
+        style.appendChild(document.createTextNode(printStyles));
+
+        document.head.appendChild(style);
+      }
+    });
+  }, [patient, printIdentifierStickerSize, headerTitle, closeModal]);
+
+  const handleAfterPrint = useCallback(() => {
+    onBeforeGetContentResolve.current = null;
+    setIsPrinting(false);
+  }, []);
+
+  const handlePrint = useReactToPrint({
+    content: () => contentToPrintRef.current,
+    documentTitle: `${patientDetails.name} - ${headerTitle}`,
+    onBeforeGetContent: handleBeforeGetContent,
+    onAfterPrint: handleAfterPrint,
+  });
+
+  const renderElementsInPairs = (elements) => {
+    const pairs = [];
+    let currentPair = [];
+
+    const getKey = (element) => {
+      if (element?.props?.children?.key?.startsWith('identifier-text')) {
+        return 'identifier';
+      }
+      return element?.key;
+    };
+
+    const filteredElements = elements.filter((element) => {
+      return printIdentifierStickerFields.includes(getKey(element));
+    });
+
+    filteredElements.forEach((element, index) => {
+      if (element) {
+        currentPair.push(element);
+        if (currentPair.length === 2) {
+          pairs.push(currentPair);
+          currentPair = [];
+        }
+      }
+    });
+
+    if (currentPair.length === 1) {
+      pairs.push(currentPair);
+    }
+
+    return pairs;
+  };
+
+  return (
+    <div>
+      <ModalHeader closeModal={closeModal} title={t('printIdentifierSticker', 'Print Identifier Sticker')} />
+      <ModalBody>
+        <div className={styles.stickerContainer} ref={contentToPrintRef}>
+          {printIdentifierStickerFields.includes('name') && (
+            <div key="name" className={styles.patientName}>
```

Comment:
```suggestion
            <div key="name" className={styles.patientName}>
```

Isn't the patient UUID more likely to be unique?

Reply:
I didn't use the patient's id because that div is only for the patient name and not for the whole patient.